src/hawkbit-client.c

```diff
@@ -537,6 +538,10 @@ gboolean install_complete_cb(gpointer ptr)
                 if (result->install_success) {
                         g_message("Software bundle installed successful.");
                         feedback(feedback_url, action_id, "Software bundle installed successful.", "success", "closed", NULL);
+                        if (hawkbit_config->post_update_reboot) {
+                                sync();
```

Comment:
Maybe it is worth executing `process_deployment_cleanup()` before syncing and rebooting to remove any leftover files that have been downloaded.

Reply:
yes, in case someone changes the `bundle_download_location` this would be useful. Thanks.